### ISSUE TITLE
Allow rebinding of look keys

### ISSUE THREAD

**Issue by megatog615** [fgsfdsfgs/perfect_dark#474]
By default the arrow keys are used to look up, down, left, and right. I would like to be able to rebind those actions and/or unbind them entirely in the ext control scheme keybindings menu so I can use those keys for other things.

**Comment by LonelySpaceDetective** [comment]
Correct me if I'm wrong, but you could just edit pd.ini in the game directory directly for this, right?

Just change the following:

> STICK_XNEG=LEFT
> STICK_XPOS=RIGHT
> STICK_YNEG=DOWN
> STICK_YPOS=UP

to:

> STICK_XNEG=NONE
> STICK_XPOS=NONE
> STICK_YNEG=NONE
> STICK_YPOS=NONE

or to whatever keys you want them to be bound to.

Admittedly though, being able to do it in-game would be more convenient.

**Comment by fgsfdsfgs** [comment]
That is correct. For a second I thought that the reason they are not rebindable in the menu is because unbinding them will cause you to lose control of the menu, but you can still control it with whatever is bound to WASD. In any case, now that there is mouse control this is not as much of a problem.

**Comment by EdHerdman** [comment]
I'd appreciate being able to use the arrow keys for movement, and probably other left-handed mouse users would as well. If the movement and look controls are being reused internally by the menu system, that seems like a good place to break with the original system as having unseen mappings makes the configuration system even less transparent to users.
